<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Script MT</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout" stretch="1,15,6,1">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="titleLayout" stretch="1,12,8,13">
       <item>
        <widget class="QPushButton" name="manualButton">
         <property name="font">
          <font>
           <family>Script MT</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Manual</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="gameTitle">
         <property name="font">
          <font>
           <family>Bell MT</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Please select the Property and Profile Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="profileLayout" stretch="1,1,1,1">
       <item>
        <layout class="QVBoxLayout" name="playerLayout_1" stretch="13,1,1,1">
         <item>
          <widget class="QGraphicsView" name="playerProfilePhoto_1"/>
         </item>
         <item>
          <spacer name="verticalSpacer_1">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addComputer_1">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Computer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPlayer_1">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Player</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="playerLayout_2" stretch="13,1,1,1">
         <item>
          <widget class="QGraphicsView" name="playerProfilePhoto_2"/>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addComputer_2">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Computer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPlayer_2">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Player</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="playerLayout_3" stretch="13,1,1,1">
         <item>
          <widget class="QGraphicsView" name="playerProfilePhoto_3"/>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addComputer_3">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Computer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPlayer_3">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Player</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="playerLayout_4" stretch="13,1,1,1">
         <item>
          <widget class="QGraphicsView" name="playerProfilePhoto_4"/>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addComputer_4">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Computer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addPlayer_4">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add Player</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="photoListLayout">
       <item>
        <widget class="QGraphicsView" name="profilePhotoList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="functionLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="commitButton">
         <property name="text">
          <string>Start!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
